lib/tasks/links_migration/link_migrator.rb

```diff
@@ -0,0 +1,45 @@
+module Tasks
+  module LinksMigration
+    class LinkMigrator
+      def populate_topic_links
+        relevant_subscriber_lists.each do |sl|
+          base_path = base_path_from(sl)
+          content_item = Services.content_store.content_item(namespaced(base_path))
+          if content_item && content_item.content_id.present?
+            puts "Updating links in SubscriberList #{sl.id}"
+            sl.update(links: {topics: [content_item.content_id]})
+          end
+        end
+      end
+
+      def destroy_non_matching_subscriber_lists
+        relevant_subscriber_lists.each do |sl|
```

Comment:
Using low-meaning variables is discouraged in [the styleguide](https://github.com/alphagov/styleguides/blob/master/ruby.md) when not on a single line.  Perhaps `list` here?